### PR TITLE
Pedantics.

### PR DESCRIPTION
**Pull request by SkyTheCodeMaster**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ Awesome libraries, programs, and mods concerning ComputerCraft and its forks.
 
 ### Charts and tables
 
-- [Cake's hex font chart](https://thox.madefor.cc/_images/encodings-cc-chars.png) and [Wojbie's decimal font chart](https://cdn.discordapp.com/attachments/477911902152949771/933498000385400862/1642633650325141456271.png): Contains decimal and hex mappings to ComputerCraft font characters.
```
Comment (tomodachi94) [inline README.md:90]: I don't think we need to separate these.

File: README.md [hunk 2]
```diff
@@ -63,7 +63,7 @@ Awesome libraries, programs, and mods concerning ComputerCraft and its forks.
 ### Emulators
 
 - CraftOS-PC: A CC:T emulator written in C++ and intends to be fast. **[Web](https://www.craftos-pc.cc) - [Source](https://github.com/MCJack123/craftos2) - [Docs](https://www.craftos-pc.cc/docs) }**
-- CCEmuX: A CC and CC:T emulator which directly uses those mods' implementations of LuaJ and the BIOS. **[Web](https://emux.cc) - [Source](https://github.com/CCEmuX/CCEmuX) - [Docs](https://emux.cc/getting-started.html)**
+- CCEmuX: A CC and CC:T emulator which directly runs the mod jars themselves. **[Web](https://emux.cc) - [Source](https://github.com/CCEmuX/CCEmuX) - [Docs](https://emux.cc/getting-started.html)**
```
Comment (tomodachi94) [inline README.md]: I think changing this is a good idea, but maybe reword it a bit:
```suggestion
- CCEmuX: A CC and CC:T emulator which executes the mods directly. **[Web](https://emux.cc) - [Source](https://github.com/CCEmuX/CCEmuX) - [Docs](https://emux.cc/getting-started.html)**
```

**Review by tomodachi94 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (SkyTheCodeMaster) [reply README.md:90]: These are not separated, just flipped around to make sense with the following text.

Comment (tomodachi94) [reply README.md:90]: @SkyTheCodeMaster why were these separated?

Comment (tomodachi94) [reply README.md:90]: Oh, alright. The diff made it look odd.

**Review by tomodachi94 — APPROVED** [review]
